Kernel/Arch/aarch64/Registers.h

```diff
@@ -134,8 +134,8 @@ struct alignas(u64) MPIDR_EL1 {
 };
 static_assert(sizeof(MPIDR_EL1) == 8);
 
-// https://developer.arm.com/documentation/ddi0595/2021-06/AArch64-Registers/ID-AA64MMFR0-EL1--AArch64-Memory-Model-Feature-Register-0
-// Memory Model Feature Register 0
+// https://developer.arm.com/documentation/ddi0601/2022-12/AArch64-Registers/ID-AA64MMFR0-EL1--AArch64-Memory-Model-Feature-Register-0
+// ID_AA64MMFR0_EL1, AArch64 Memory Model Feature Register 0
```

Comment:
You can keep this commit as-is, but could you merge all of the `Add CPU Register` and `Read CPU Feature` commits into one mega-commit for `Add and Read Aarch64 CPU feature registers` ?

Reply:
will do.